test/integration/src/main/java/io/pravega/test/integration/demo/ControllerWrapper.java

```diff
@@ -102,6 +103,21 @@ public ControllerWrapper(final String connectionString, final boolean disableEve
                              boolean enableAuth, String passwordAuthHandlerInputFilePath,
                              String tokenSigningKey, boolean isRGWritesWithReadPermEnabled,
                              int accessTokenTtlInSeconds) {
+        this (connectionString, disableEventProcessor, disableControllerCluster, controllerPort, serviceHost,
+                servicePort, containerCount, restPort, enableAuth, passwordAuthHandlerInputFilePath, tokenSigningKey,
+                isRGWritesWithReadPermEnabled, accessTokenTtlInSeconds, false, "", "", "", "", false);
+    }
+
+    @Builder
```

Comment:
Does this @Builder generate a seperate builder?

Reply:
It generates a builder for the `ControllerWrapper` based on the parameters used in this Constructor (as opposed to the `Class-level` annotation we use in the `ClusterWrapper`. The idea was to avoid interface changes to `ControllerWrapper` so that the many tests that use it don't need to change.